website/README.md

```diff
@@ -1,49 +1,161 @@
+documentation.
+In that case, once you have made the changes to the `main` docs under `docs/`,
+copy the changes over to the correct version in `versioned_docs/`. If you need
+to change the sidebar at that point, change the correct version in
+`versioned_sidebars`.
+
+#### How to change the navbar
+
+The navbar is controlled in `docusaurus.config.js`. It is important to note that
+this is not (and cannot be) versioned, meaning that if you change the navbar
+that change will be present for all versions.
+
+#### How to add yaml/json/console output without making it painful
+
+Our product comes with a lot of `yaml`, and we need to document that. However,
+too much can saturate a page and make it hard to read. It can also be easy to
+miss bits of text lying in between large chunks of `yaml`. The solution is to
+collapse large `yaml`, `json` or console output sections with `<details>` tags.
+
+Smaller chunks are usually fine. But for anything more that 10 lines it is
+neater to wrap it.
+
+~~~markdown
+<details><summary>Expand to see this awesome HelmRepository</summary>
+
+Don't forget to leave a line break below the opening tag...
+```yaml
+---
+woohoo:
+  such: yaml
 ```
+and before the closing one.
 
-This command starts a local development server and opens up a browser window. To preview your changes as you edit the files, you can run a local development server that will serve your website and reflect the latest changes.
+</details>
+~~~
 
-If you are using newer version of Node (17+):
+<details><summary>Expand to see this awesome HelmRepository</summary>
 
-```bash
-export NODE_OPTIONS=--openssl-legacy-provider
+Don't forget to leave a line break below the opening tag...
 ```
+---
+woohoo:
+  such: yaml
+```
+and before the closing one.
+
+</details>
+
+#### How to add screenshots
+
+Please, please, zoom in before you take the shot. I promise they don't need to
+see the whole page, just the bit you are talking about is fine.
+
+### After you have finished your changes
 
-## Build
+Check that the docs build with:
+
+```bash
+export GA_KEY=fakekey
 
-```console
 yarn build
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
-During development once you're happy with your changes, you should run this command to build the site locally to make sure all your changes are good.
-This command can be bit slow and may take some time to build the site. 
+The build will also be run on the PR, but it is convenient to run it before you
+open one.
+
+## Deploying the docs
 
-## Deployment
+Do not attempt to deploy the docs manually.
 
-Deployment happens automatically (once the tests pass) upon merging to the default branch: see [.github/workflows/docs.yaml](.github/workflows/docs.yaml) for config.
+Deployment happens automatically (once the tests pass) upon merging to the
+default branch: see [.github/workflows/docs.yaml](.github/workflows/docs.yaml) for config.
```

Comment:
```suggestion
`main` branch: see [.github/workflows/docs.yaml](.github/workflows/docs.yaml) for config.
```
more explicit so no one has to make up their minds about what the default branch is.